src/pages/Player.vue

```diff
@@ -244,6 +276,17 @@ export default {
         }
       }
     },
+    itemResponses: {
+      handler(value) {
+        // whenever itemResponses update, re-render the markers
+        // and re-calculate the scorecard metrics
+        if (value != undefined && this.player != undefined) {
+          this.showItemMarkersOnSlider(this.player);
+          this.calculateScorecardMetrics();
+        }
```

Comment:
why do we need to re-render markers? is it because of the bug we saw earlier?
can't the scoreboard metrics be calculated reactively?